docs/kratos/concepts/rest-api.mdx

```diff
@@ -1,48 +0,0 @@
----
-id: rest-api
-title: REST API Design
----
-
-This chapter contains generally applicable information on API design. When using Ory services, one can expect a consistent
-experience when interacting with REST APIs.
-
-## Pagination
```

Comment:
What's the replacement for this? Pagination is a common question

Reply:
https://www.ory.sh/docs/ecosystem/api-design#pagination